tests/test_data_collections.py

```diff
@@ -89,25 +110,37 @@ def test_attributes() -> None:
     assert data_collection.service_url is None
 
 
-def test_sentinel1_checks() -> None:
-    assert DataCollection.SENTINEL1_IW.is_sentinel1
-    assert not DataCollection.SENTINEL2_L1C.is_sentinel1
-
-    assert DataCollection.SENTINEL1_IW_ASC.contains_orbit_direction("ascending")
-    assert not DataCollection.SENTINEL1_IW_DES.contains_orbit_direction("ascending")
-
-    assert DataCollection.SENTINEL2_L2A.contains_orbit_direction("descending")
+@pytest.mark.parametrize(
+    "test_collection, expaected",
+    [
+        (DataCollection.SENTINEL2_L1C, False),
+        (DataCollection.SENTINEL1_EW, True),
+        (DataCollection.LANDSAT_TM_L1, False),
+    ],
+)
+def test_is_sentinel1(test_collection: DataCollection, expaected: bool) -> None:
```

Comment:
correct `expaected` (also in following tests)

Reply:
done